doc/Remove-Account.md

```diff
@@ -11,13 +11,13 @@ with your web browser.
 You will need to be logged in at the time.
 
 You will be asked for your password to confirm the request.
-If this matches your stored password, your account will immediately be blocked to all probing.
-Unlike some social networks we do **not** hold onto it for a grace period in case you change your mind.
-All your content and user data, etc is instantly removed.
-For all intents and purposes, the account is gone in moments.  
-
-We then send out an "unfriend" signal to all of your contacts.
-This signal deletes all content on those networks.
-Unfortunately, due to limitations of the other networks, this only works well with Friendica contacts.
-We allow four days for this, in case some servers were down and the unfriend signal was queued.
-After this, we finish off deleting the account.
+If this matches your stored password, your account will immediately be marked as deleted.
+There is no grace period, this action cannot be reverted.
+Most of your content and user data will be deleted on short term.
```

Comment:
"Most of your user data will be deleted immediately.
Your content will be gradually deleted over time."

Reply:
It isn't deleted immediately. It is done via a worker process with the lowest priority. So it can take hours before the deletion even starts.

Reply:
Ok, so "Most of your content and user data will be deleted shortly in the background."